SmoochHelpKit.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.frameworks            = "SystemConfiguration", "UIKit", "Foundation", "OpenGLES", "QuartzCore", "CoreText"
   s.library               = "xml2"
   s.xcconfig              = { "HEADER_SEARCH_PATHS" => "$(SDKROOT)/usr/include/libxml2" }
-  s.module_map            = "SmoochHelpKit/libxml/module.modulemap" 
-  s.dependency "Smooch"
-end
\ No newline at end of file
+  s.module_map            = "SmoochHelpKit/libxml/module.modulemap"
+  s.dependency "Smooch", "> 6.0"
```

Comment:
This is my first time editing a Podspec, hopefully I got this right!

Reply:
I don't think this will work since this podspec is specifying a platform of iOS 7.0, but Smooch 6.x supports a minimum of 8.0

Maybe revert this line, and change the platform target to 8.0 instead?